index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Awesome Books</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <script src="https://code.jquery.com/jquery-3.5.1.min.js"></script>
+    <script type="module" src="js/index.js"></script>
+  </head>
+
+  <body>
+    <section class="header">
+        <h1>Awesome Books</h1>
+        <nav class="navbar">
+          <ul>
+            <li><a id="listbutton" href="#">List</a></li>
+            <li><a id="addnewnavbutton" href="#">Add New</a></li>
+            <li><a id="contactbutton" href="#">Contact</a></li>
+          </ul>
+        </nav>
+    </section>
+    <section id="date-time" class="date-time"></section>
```

Comment:
As specified in the [requirements](https://github.com/microverseinc/curriculum-javascript/blob/17555a4b0793b5065e3aeda6d2568fddd9c45330/books/books_with_es6.md?plain=1#L52), kindly ensure that the current date is displayed below the navigation bar.

![image](https://user-images.githubusercontent.com/20554583/235421512-5534fe56-0cf7-4939-91e6-0671e427fcf8.png)